### PR TITLE
Setup project for python.

### PR DESCRIPTION
**Pull request by zidder**
Add `setup.sh` shell file, requirements for python and visualization script.
Update README.md

### CODE REVIEW

File: bin/setup.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#! /bin/bash
+
+suto apt-get install python3.7
```
Comment (altunyanv) [inline bin/setup.sh]: `sudo ...`

**Comment by salbert96** [comment]
@zidder I believe it would be good to have all the files inside a separate folder, so that we can have other  projects inside the same repo? Also I think it would be great to have a docker for the setup of the environment. Ruben can help you with that.

**Comment by salbert96** [comment]
@zidder I will review this one and add docker based approach

**Comment by salbert96** [comment]
@zidder I added docker based setup scripts. But it is better to use it only for deployment purposes since for visualization you need to connect the display.

**Pull request merged**